docker/centos7/Dockerfile

```diff
@@ -441,6 +441,11 @@ RUN if [ "$(uname -m)" == "aarch64" ]; then \
     ./rustup-init -y --no-modify-path --profile minimal --default-toolchain ${RUST_VERSION} --default-host ${RUST_ARCH} && \
     rm -rf /tmp/*
 
+# build/install Intel Compiler(ICX/ICPX)
+RUN curl -LsO https://registrationcenter-download.intel.com/akdlm/irc_nas/18679/l_HPCKit_p_2022.2.0.191_offline.sh && \
+    sh ./l_HPCKit_p_2022.2.0.191_offline.sh -a -s --eula accept && \
```

Comment:
what is in this EULA? Something about this change makes me think I would need to consult a legal department.. I'm willing to be wrong, but at the moment I'm not sure about this.

Reply:
Hi! The latest EULA can be found [here](https://www.intel.com/content/www/us/en/developer/articles/license/end-user-license-agreement.html#collapseCollapsible1637778017389). Did you have any discussion with the legal department?